src/displayapp/screens/Steps.h

```diff
@@ -32,12 +32,15 @@ namespace Pinetime {
         uint32_t currentTripSteps = 0;
 
         lv_obj_t* lSteps;
+        lv_obj_t* lStepsYesterday;
         lv_obj_t* stepsArc;
         lv_obj_t* resetBtn;
         lv_obj_t* resetButtonLabel;
         lv_obj_t* tripLabel;
 
-        uint32_t stepsCount;
+        const char* yesterdayStr;
```

Comment:
Could yesterdayStr be a constant?
If that's fine, you could stick it in an anonymous namespace in the implementation as `constexpr const char *` (I think!) and then it will live in just flash rather than flash and RAM

Reply:
In theory that's how it should work right now too, because I'm not copying the string, just taking the address of a string literal (that's why I don't need to free it either)

Unless I'm misunderstanding something

Reply:
It doesn't matter here too much as it's heap memory rather than stack, but you can avoid storing the pointer at all by effectively inlining the constant